satpy/composites/sar.py

```diff
@@ -38,6 +38,16 @@ def overlay(top, bottom, maxval=None):
     return res.clip(min=0)
 
 
+def soft_light(top, bottom, maxval):
+    """Apply soft light.
```

Comment:
Since I asked for other changes, maybe a little more descriptive on this docstring? Eh I guess you have the link.

Reply:
I wouldn't know how to describe the effect actually

Reply:
Good enough answer for me. :grin: